webapp/sources/rudder/rudder-core/src/main/resources/hooks.d/policy-generation-finished/60-trigger-node-update.properties

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
This is mades to be sourced, there should be no shebang and no exec bit

Reply:
done